City-of-Helsinki/drupal-hdbt
# UHF-2766: Make the react and share block description color accessible

To test:

1. `composer require drupal/hdbt:dev-UHF-2766_react_and_share_accessibility_fix`
2. `make drush-cr`
3. Make sure that the description text on react and share block is accessible now (should have opacity 1). If you answer no or only somewhat to the block you see the description text.
